### PR TITLE
Add topics collection method to openrtb

### PR DESCRIPTION
**Pull request by patmmccann**
We may want to know if it came from a header or was observed by a publisher

### CODE REVIEW

File: proposals/topics-rtb/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ OpenRTB `BidRequest.user.data` can be used to represent Topics.
 | ------------ | ----------- | ----------- |
 | `segtax`  | Integer       | A taxonomy enumerated in the [segtax extension](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/segtax.md) |
 | `segclass`   | String        |  Classifier version, as provided by the browser |
+| `method` | Enum | 1: from window.browsing.topics(), 2: from a request header |
```
Comment (sbelov) [inline proposals/topics-rtb/README.md:24]: Why would this distinction – accessing topics through window.browsing.topics() versus a request header – be useful to the downstream buyers?

Wouldn't the set of topics from either mechanism be identical?

Comment (patmmccann) [reply proposals/topics-rtb/README.md:24]: It seems the latter may be more trusted by buyers

Comment (sbelov) [reply proposals/topics-rtb/README.md:24]: OOC, why would the latter be more trusted by buyers?
Additionally, when one receives a bid request… isn't there an inherent level of trust to the data received in such a bid request?

Comment (patmmccann) [reply proposals/topics-rtb/README.md:24]: We believe a DSP might trust their SSP partners discoveries over the publisher assertions.

**Pull request closed without merging**